data/schedule.json

```
@@ -1,100 +1,66 @@
 [
   {
     "day": "1",
-    "date": "Oct. 10th, Thursday",
+    "date": "October 9, 2025",
     "mainStage": [
-      { "time": "9:00 - 10:00", "title": "Doors open & breakfast", "isBreak": true },
-      { "time": "10:00 - 10:30", "talk": "through-the-fire-and-the-flames" },
-      {
-        "time": "10:45 - 11:15",
-        "talk": "build-bigger-in-less-time-code-testing-beyond-the-basics"
-      },
-      {
-        "time": "11:30 - 12:00",
-        "talk": "a-gentle-introduction-to-procedural-macros"
-      },
-      { "time": "12:00 - 13:30", "title": "Lunch", "isBreak": true },
-      { "time": "13:30 - 14:00", "talk": "practical-rust-for-web-audio" },
-      {
-        "time": "14:15 - 14:45",
-        "talk": "augmented-docs-a-love-letter-to-rustdoc-and-docsrs"
-      },
-      {
-        "time": "15:00 - 15:30",
-        "talk": "the-impact-of-memory-allocators-on-performance-a-deep-dive"
-      },
-      { "time": "15:30 - 16:00", "title": "Snack Break", "isBreak": true },
-      { "time": "16:00 - 16:30", "talk": "proving-macro-expansion-with-expandable" },
-      {
-        "time": "16:45 - 17:15",
-        "talk": "runtime-scripting-for-rust-applications"
-      },
-      { "time": "17:30 - 18:00", "talk": "unleashing-rust-the-ferris-within" }
+      { "time": "9:00–10:00", "title": "Doors open & breakfast", "isBreak": true },
+      { "time": "10:00–10:45", "title": "Opening talk, TBA", "isBreak": true },
+      { "time": "11:00–11:30", "talk": "data-engineering-with-rust" },
+      {"time": "11:45–12:15",  "talk": "from-any-to-this" },
+      { "time": "12:15–14:00", "title": "Lunch", "isBreak": true },
```

```suggestion
      { "time": "12:15–13:45", "title": "Lunch", "isBreak": true },
```

almost 2h of lunch is too long